src/replit_graphql.rs

```diff
@@ -275,17 +284,53 @@ impl ProfileRepls {
                     errored.push(repl.id);
                     progress.failed.failed += 1;
                 }
-                Ok(Ok(_)) => {
+                Ok(Ok(DownloadStatus::NoHistory)) => {
+                    info!(
+                        "Downloaded {}::{} (without history) to {}",
+                        repl.id, repl.slug, download_zip
+                    );
+                    no_history_download_count += 1;
+                    progress.failed.no_history += 1;
+
+                    if let Err(err) = fs::remove_dir_all(download_locations.git).await {
+                        warn!(
+                            "Error removing git temp dir for {}::{}: {err}",
+                            repl.id, repl.slug
+                        )
+                    }
+
+                    if let Err(err) = fs::remove_dir_all(download_locations.main).await {
+                        warn!(
+                            "Error removing main temp dir for {}::{}: {err}",
+                            repl.id, repl.slug
+                        )
+                    }
+
+                    if let Err(err) = fs::remove_dir_all(download_locations.ot).await {
+                        warn!(
+                            "Error removing ot temp dir for {}::{}: {err}",
+                            repl.id, repl.slug
+                        )
+                    }
+
+                    if let Err(err) = fs::remove_dir_all(download_locations.staging_git).await {
+                        warn!(
+                            "Error removing git staging temp dir for {}::{}: {err}",
+                            repl.id, repl.slug
+                        )
+                    }
```

Comment:
sick i love the err handing instead of ignoring them - wondering if we can just do something like

```rust
fs::delete_all(vec![dir1, dir2])
```
then `error!` with the dir that failed? i don't think that's a buitin though :(( (don't bother making one)

Reply:
I don't believe it is a builtin sadly.